src/ResourceManager/Kubernetes/Commands.Kubernetes/load.ps1

```diff
@@ -0,0 +1,3 @@
+Import-Module -Name ./src/Package/Release/ResourceManager/AzureRM.Profile.Netcore/AzureRM.Profile.Netcore.psd1
```

Comment:
??

Reply:
This was just for testing. I'll drop it before the PR is final.